### DISCUSSION TITLE
Get mouse position on the renderer? Something changed!

### DISCUSSION THREAD

**Discussion by ghost** [pixijs/pixijs#8975] [Q&A] [answered]
So before, it was easy to get the mouse position on the renderer (not the stage). It used to be:

`app.renderer.plugins.interaction.on('pointermove', (e) => {
  Mouse.pos = e.data.global;
});`

I can't for the life of me figure it out in PIXI v7. Can someone please clear this up?

**Comment by SuperSodaSea** [comment, marked as answer]
`@pixi/interaction` is removed in PixiJS v7, and with the new `@pixi/events` you can only listen on a `DisplayObject`. If you want to get the mouse position for the whole canvas, you can listen to the DOM event directly:
```js
app.view.addEventListener('pointermove', e => {
    console.log(e.clientX, e.clientY);
});
```

**Comment by ghost** [comment]
Hi and thanks for the input. I did try that but because of the mechanics of my game, I ended up going with this:

```
window.addEventListener('pointermove', (e) => {
  let cRect = app.view.getBoundingClientRect();
  Mouse.pos.x = e.clientX - cRect.left;
  Mouse.pos.y = e.clientY - cRect.top;
})
```
